app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="library">Biblioteca de Sons</string>
+  <string name="about">Saiba mais</string>
+  <string name="report_issue">Informar sobre problema</string>
+  <string name="repeat_time_period">Período até repetir som</string>
+  <string name="volume">Volume do som</string>
+  <string name="play_pause">tocar ou pausar som</string>
+  <string name="open_drawer">Abrir menu de navegação</string>
+  <string name="close_drawer">Fechar menu de navegação</string>
+  <string name="notification_channel_default__name">Reprodução de audio</string>
+  <string name="notification_channel_default__description">Controles de audio</string>
+  <string name="pause">Pausar</string>
+  <string name="play">Tocar</string>
+  <string name="stop">Parar</string>
+  <string name="airplane_inflight">Avião voando</string>
+  <string name="airplane_seatbelt_beeps">Anúncio em avião</string>
+  <string name="birds">Pássaros</string>
+  <string name="bonfire">Fogueira</string>
+  <string name="brownian_noise">Ruído vermelho (Brownian)</string>
```

Comment:
I was playing with this and Google corrected me to "Ruído Browniano". Maybe this is more suited?

Reply:
Not sure which one is more suitable for the context, but it seems that both translations are valid. Quoting [Ruído marrom](https://pt.wikipedia.org/wiki/Ru%C3%ADdo_marrom) in Wikipédia:  "ruído marrom, ruído vermelho ou ruído browniano" (three options).

Reply:
When translating this, I took as reference both the French translation for the app and that Wikipedia article and came to the conclusion that "ruído vermelho" (red noise) was the best suited here, and included "browniano" in parenthesis for clarification's sake.
However, "Brownian" instead of of "browniano" was indeed a typo on my end >.<